Kitodo-API/src/main/java/org/kitodo/api/dataformat/LinkedStructure.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat;
+
+import java.math.BigInteger;
+import java.net.URI;
+
+/**
+ * A link to a structure in a different {@code Workpiece}.
+ */
+public class LinkedStructure implements ExistingOrLinkedStructure {
+
+    /**
+     * The label of the linked structure. The label is saved in the linked file.
+     */
+    private String label;
+
+    /**
+     * The order is a number that serves to place other structures together with
+     * them in the correct order in the same parental structure. After a link
+     * has been made, the order value no longer affects the order. The order is
+     * saved in the file that contains the link.
+     */
+    private BigInteger order;
+
+    /**
+     * The type of structure, for example, book, chapter, page. Although the
+     * data type of this variable is a string, it is recommended to use a
+     * controlled vocabulary. If the generated METS files are to be used with
+     * the DFG Viewer, the list of possible structure types is defined. The type
+     * is saved in the linked file.
+     * 
+     * @see "https://dfg-viewer.de/en/structural-data-set/"
+     */
+    private String type;
+
+    /**
+     * The URI of the linked process. The URI is saved in the file that contains
+     * the link.
+     */
+    private URI uri;
+
+    @Override
+    public String getLabel() {
+        return label;
+    }
+
+    /**
+     * Returns the order value of the linked structure.
+     * 
+     * @return the order value
+     */
+    public BigInteger getOrder() {
```

Comment:
It should be arbitrary type-in. Only big integer is arbitrary. Otherwise you have a secret limit that the person typing needs to now, keep in mind and plan with. So I would like to leave it this way here.